### ISSUE TITLE
Show the organization's brand logo in the main app navbar

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#3842]
Organizations presenting or embedding Lightdash want the main app navbar to show their own logo — "so that when I'm presenting dashboards, sharing screens etc, things feel a bit more branded". Org settings → Appearance already stores brand logos, colours and fonts, but nothing consumes the stored logos yet and the logo section of the settings UI is disabled. First release: render the stored organization logo in the main app navbar and re-enable the logo settings UI.

## Impact

* White-label P0 for an enterprise prospect (self-hosted, no egress) embedding the full Lightdash workspace; a second customer asked for exactly this and "could not see anything obvious" in settings
* No workaround: the navbar logo is hardcoded

## Open decisions

* **Logo delivery on no-egress self-hosted**: the brand store is URL-based and Brandfetch auto-fetch requires egress. The no-egress prospect must set the logo manually and says they need file upload (they're still confirming whether an image URL reachable inside their network would do). Ship v1 with manual URL entry only, or add logo file upload + image storage?
* **Navbar colour**: the same prospect confirmed a logo swap alone isn't enough — the navbar theme colour matters too, and brand colours are already stored. Apply a stored brand colour to the navbar in v1, or keep v1 logo-only with colours tracked in [#5370](<https://github.com/lightdash/lightdash/issues/5370>)?

## Scope

* This issue: navbar logo + re-enabled logo settings UI (plus navbar colour, pending the decision above)
* Follow-up issue (link to follow): custom navbar links and per-tenant branding for embedded use cases — the prospect driving this confirmed a single org-wide brand is enough for their first release
* Tracked elsewhere: broader appearance controls (fonts, page/tile backgrounds) in [#5370](<https://github.com/lightdash/lightdash/issues/5370>); email branding in [#13351](<https://github.com/lightdash/lightdash/issues/13351>); Slack/Teams delivery and app-identity branding to be scoped separately
* Login page: out of scope for now (the requesting customer handles sign-in via their own user sync)

# Additional agent context

## Where to start

* `packages/frontend/src/components/NavBar/MainNavBarContent.tsx` — renders a hardcoded `Logo` imported from `svgs/logo-icon.svg` (a square icon mark); this is the swap point
* `packages/frontend/src/components/NavBar/index.tsx` — the navbar is forced dark (`forceColorScheme="dark"`), so render the stored variant meant for dark backgrounds. `OrganizationBrandLogo` in `packages/common/src/types/organizationBrand.ts` documents that `theme` describes the artwork colour (`'light'` = light/white logo for dark backgrounds) and `type` distinguishes `logo`/`symbol`/`icon`
* `packages/frontend/src/components/UserSettings/AppearanceSettingsPanel/BrandAppearanceSettings.tsx` — logo tiles are commented out ("TODO: re-enable logo tiles once logos have a consumer"); `packages/frontend/src/hooks/organization/useOrganizationBrand.ts` has the fetch/save hooks
* Prior art: lightdash/lightdash#25303 shipped Brand Appearance settings, including a manual-edit save path (`SaveOrganizationBrandRequest`) that persists values without calling Brandfetch; auto-fetch is optional, gated on `BRANDFETCH_API_KEY` (`packages/backend/src/config/parseConfig.ts`)

## Verifications

* An org with a stored brand logo sees it in the main app navbar in place of the Lightdash logo, legible against the dark navbar
* An org with no stored brand logo sees the current navbar unchanged
* Logos can be viewed and edited in Appearance settings on a deployment with no `BRANDFETCH_API_KEY` configured
* Dashboard-only embeds stay chrome-free, unchanged

**Comment by 12ian34** [comment]
also known as "white labelling" or "whitelabelling". adding for searchability

**Comment by tonytusharjr** [comment]
I would like this option too. Is it any easier on storage if the Lightdash config required the user to setup a cloud storage/s3 bucket for holding branding assets?

**Comment by TuringLovesDeathMetal** [comment]
Cycle did this automatically when I created an org, and it was such a nice touch! More info in Slack here: https://lightdash.slack.com/archives/C02DGV9RWKV/p1714126806277959

**Comment by 12ian34** [comment]
related to #5370

**Comment by PriPatel** [comment]
A user has requested the ability to have 'branded navbars' in dashboard which include logos and links. These navbars should be able to be consistent applied everywhere across the app.

**Comment by ilhamfajaarr** [comment]
Hello does anyone still has the documentation for this? i cant open the slack ones

**Comment by torimwhaley** [comment]
Heads up for watchers — we've re-scoped this issue to a concrete first release: show your organization's logo in the main app navbar (see the updated issue description). Branded navbars with custom links, and per-tenant branding for embedded use cases, will be scoped as a separate issue with a link to follow. Broader appearance controls stay tracked in [#5370](<https://github.com/lightdash/lightdash/issues/5370>) and email branding in [#13351](<https://github.com/lightdash/lightdash/issues/13351>). If this narrowing drops context you care about, shout here.

**Comment by jpetey75** [comment]
Organization brand settings shipped, so you can store and edit your logo, colors, and fonts in appearance settings ([#25303](https://github.com/lightdash/lightdash/pull/25303)), and report emails can be whitelabeled from your own domain on Cloud ([#25268](https://github.com/lightdash/lightdash/pull/25268)). The navbar itself still shows the Lightdash logo, so this issue stays open for navbar branding.
